### DISCUSSION TITLE
question about a word 'future complete'

### DISCUSSION THREAD

**Discussion by mreraser** [huggingface/diffusers#8278] [Q&A] [answered]
Hello! I have a question about the word 'future complete' in this sentence. 

[Philosophy - Pipeline](https://huggingface.co/docs/diffusers/v0.27.2/en/conceptual/philosophy#pipelines)
8th bullet point 

> - Pipelines are not intended to be feature-complete user interfaces. For **future complete** user interfaces one should rather have a look at [InvokeAI](https://github.com/invoke-ai/InvokeAI), [Diffuzers](https://github.com/abhishekkrthakur/diffuzers), and [lama-cleaner](https://github.com/Sanster/lama-cleaner).

Is this meaning 'user interface that will be be complete in future'?

**Comment by tolgacangoz** [comment]
Hmm, it sounds like a typo to me. IMO, "future complete" might be "feature-complete" :thinking:.
Cc: @stevhliu

**Reply by mreraser** [reply]
Thank you for your reply👍! I think so too.

**Reply by stevhliu** [reply, marked as answer]
Yeah this is a typo! 😅 Would you like to open a PR to correct it to "feature-complete"?

**Reply by mreraser** [reply]
> Yeah this is a typo! 😅 Would you like to open a PR to correct it to "feature-complete"?

Yes, I understand. I'll open a PR shortly. Thank you very much for the review.

**Reply by mreraser** [reply]
Also thank you very much @tolgacangoz !
